app/components/templates/form.tsx

```diff
@@ -0,0 +1,258 @@
+          defaultValue={name || ""}
+          placeholder="Booking Arrangement 2023"
+          required={zodFieldIsRequired(NewTemplateFormSchema.shape.name)}
+        />
+      </FormRow>
+      <FormRow
+        rowLabel="Type"
+        className="border-b-0 pb-0"
+        required={zodFieldIsRequired(NewTemplateFormSchema.shape.type)}
+      >
+        <Select
+          name={zo.fields.type()}
+          defaultValue={selectedType}
+          disabled={disabled}
+          onValueChange={(val) =>
+            setSelectedType(
+              val.toString() === TemplateType.BOOKINGS
+                ? TemplateType.BOOKINGS
+                : TemplateType.CUSTODY
+            )
+          }
+        >
+          <SelectTrigger
+            disabled={isEdit}
+            className="px-3.5 py-3"
+            placeholder="Choose a field type"
+          >
+            <SelectValue />
+          </SelectTrigger>
+          <SelectContent
+            position="popper"
+            className="w-full min-w-[300px]"
+            align="start"
+          >
+            <div className=" max-h-[320px] overflow-auto">
+              {[TemplateType.BOOKINGS, TemplateType.CUSTODY].map((value) => (
+                <SelectItem value={value} key={value}>
+                  <span className="mr-4 text-[14px] text-gray-700">
+                    {value}
+                  </span>
+                </SelectItem>
+              ))}
+            </div>
+          </SelectContent>
+        </Select>
+      </FormRow>
+      <FormRow
+        required={zodFieldIsRequired(
+          NewTemplateFormSchema.shape.signatureRequired
+        )}
+        rowLabel="Ask for signature"
+        subHeading={<p>Users will be asked to sign document</p>}
+        className="border-b-0 pb-0"
+      >
+        <Switch
+          name={zo.fields.signatureRequired()}
+          disabled={disabled}
+          defaultChecked={signatureRequired}
+        />
+      </FormRow>
+      <FormRow
+        className="border-0 pb-0"
+        rowLabel="Description"
+        required={false}
+      >
+        <Input
+          label="Description"
+          hideLabel
+          inputType="textarea"
+          name={zo.fields.description()}
+          disabled={disabled}
+          error={zo.errors.description()?.message}
+          className="w-full border-b-0 pb-0"
+          defaultValue={description || ""}
+          placeholder="Store the booking arrangement for 2023"
+          required={false}
+        />
+      </FormRow>
+      {/* <Input
+        label="Description"
+        hideLabel
+        hidden
+        inputType="textarea"
+        name={zo.fields.pdfSize()}
+        disabled={disabled}
+        className="w-full border-b-0 pb-0"
+        value={size}
+        placeholder="Store the booking arrangement for 2023"
+        required={zodFieldIsRequired(NewTemplateFormSchema.shape.pdfSize)}
+      /> */}
+      <FormRow rowLabel="Upload PDF">
+        <FileDropzone
```

Comment:
I think we might have to drop that. Its not working as expected and it will be so much easier if we just use a normal file input, like we do in most places. I know this was in the design but thats an oversight on my side that I didn't let the designer know it should be changed.